g4f/Provider/bing/create_images.py

```diff
@@ -13,10 +13,10 @@
 from typing import Generator, List, Dict
 
 from ..create_images import CreateImagesProvider
-from ..helper import get_cookies, get_event_loop
+from ..helper import get_cookies
 from ...webdriver import WebDriver, get_driver_cookies, get_browser
 from ...base_provider import ProviderType
-from ...image import format_images_markdown
+from ...image import ImageResponse
```

Comment:
Consider importing the `ImageResponse` class from the `...image` module directly, instead of importing the entire `...image` module.